web/src/copy_and_paste.ts

```diff
@@ -433,6 +433,36 @@ export function paste_handler_converter(paste_html: string): string {
             return "~~" + content + "~~";
         },
     });
+    turndownService.addRule("latexMath", {
+        filter(node: Node) {
+            if (node instanceof Element) {
+                const closestDisplay = node.closest(".katex-display");
+                const hasMathML = node.querySelector(".katex-mathml") !== null;
```

Comment:
Don't use camelCase variable names; that's not our convention.

I'm also not sure why this does the `closest` pattern rather than just processing the `katex` element at the root. For one thing, I think that means it won't work with non-display math inline KaTeX blocks? 

In any case, I pushed a commit with a proposed reworking of this that feels simpler, using the early return pattern and asserts to nest less deeply. It doesn't seem to work; can you debug?